### PR TITLE
Clay sums attempt4

### PR DESCRIPTION
**Pull request by Captainlearyo**

### CODE REVIEW

**Review by JorjaF — APPROVED** [review]
I am glad we finally got this one on here,

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -94,5 +94,41 @@ def goals_per_team
     team_goals
   end
 
+  def highest_sum 
+    hash = {}
+    @game_factory.games.each do |i|
```
Comment (andilovetto) [inline lib/stat_tracker.rb]: i think this looks great, clay! i think we should consider renaming the block variable to be more explicit as to what it represents and maybe also renaming the hash variable to be more explicit as well. 

maybe you could change the `i` to be `game` instead?

**Review by andilovetto — CHANGES_REQUESTED** [review]
i think for coder empathy and for the sake of readability, we should consider renaming the block variable `i` and the `hash` variable to something more explicit as to what those individual variables represent.

**Comment by Captainlearyo** [comment]
i was changed to game

**Comment by ChrisAsercion** [comment]
I checked and the block variables are changed to game. Thanks Clay!

**Review by ChrisAsercion — APPROVED** [review]
block variables changed to game on request has been made.

**Review by andilovetto — APPROVED** [review]
thanks for making the requested change. i still think we could change the hash variable to be more explicit as to what that hash is holding, but since you stated it was a placeholder, i think it's okay for now.

**Pull request merged**